File: programacao/Ex32.c
#include<stdio.h>
int main()
{
	int n,i,num,soma;
	printf ("Digite um numero positivo menor que 100: ");
	scanf ("%i", &n);
	while ((n<0) || (n>100))
	{
		printf ("Invalido! Insira um numero positivo menor que 100:");
    	scanf ("%i", &n);
	}	
	num=2;
	soma=3;
	for (i=1; i<=n; i++)
	{
		printf ("%i\n",num);		
		num=num+soma;
		soma=soma+2;
	}	
return 0;
}

File: python/Ex32.py
n = int(input("Digite um numero positivo menor que 100: "))
while (n<0 or n>100):
    n = int(input("Invalido! insira um numero positivo menor que 100: "))    
num=2
soma=3
for i in range (1, n, 1):
    print (num)
    num=num+soma
    soma=soma+2